webapp/channels/src/components/status_icon_new.tsx

```diff
@@ -13,14 +13,14 @@ const StatusIconNew = ({className = '', status = ''}: Props) => {
         return null;
     }
 
-    let iconName = 'icon-circle-outline';
-    if (status === 'online') {
-        iconName = 'icon-check-circle';
-    } else if (status === 'away') {
-        iconName = 'icon-clock';
-    } else if (status === 'dnd') {
-        iconName = 'icon-minus-circle';
-    }
+    const statusToIconMap: { [key: string]: string } = {
```

Comment:
Move the map to outside of the component, so we don't have to instantiate it every render.

Reply:
ok